### PR TITLE
[BH-2036] Fix missing label in the countdown to the start of meditation

### PR DESCRIPTION
**Pull request by Maciej-Mudita**
The meditation start countdown screen will be consistent with the meditation screen.

**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [ ] Has unit tests if possible
- [ ] Has documentation updated
- [ ] Has changelog entry added

### CODE REVIEW

File: products/BellHybrid/apps/application-bell-meditation-timer/windows/MeditationCountdownWindow.cpp [hunk 1]
```diff
@@ -79,6 +79,19 @@ namespace gui
         timer->setMargins(gui::Margins(0, countdownStyle::timer::marginTop, 0, 0));
         timer->setAlignment(Alignment(Alignment::Horizontal::Center, Alignment::Vertical::Center));
 
+        bottomDescription = new gui::TextFixedSize(
```
Comment (Lefucjusz) [inline products/BellHybrid/apps/application-bell-meditation-timer/windows/MeditationCountdownWindow.cpp:82]:
```suggestion
        bottomDescription = new TextFixedSize(
```

It's already in `namespace gui`, so it's not needed here :wink:

File: products/BellHybrid/apps/application-bell-meditation-timer/windows/MeditationCountdownWindow.hpp [hunk 2]
```diff
@@ -29,10 +29,11 @@ namespace gui
...
         std::unique_ptr<app::meditation::MeditationCountdownContract::Presenter> presenter;
-        VBox *mainVBox                = nullptr;
-        Arc *progress                 = nullptr;
-        TimeMinuteSecondWidget *timer = nullptr;
-        Text *description             = nullptr;
+        VBox *mainVBox{nullptr};
+        Arc *progress{nullptr};
+        TimeMinuteSecondWidget *timer{nullptr};
+        Text *description{nullptr};
+        gui::TextFixedSize *bottomDescription{nullptr};
```
Comment (Lefucjusz) [inline products/BellHybrid/apps/application-bell-meditation-timer/windows/MeditationCountdownWindow.hpp:36]: Same here

**Review by Lefucjusz — APPROVED** [review]

File: products/BellHybrid/apps/application-bell-meditation-timer/windows/MeditationCountdownWindow.cpp (hunk 1 above)
Comment (Maciej-Mudita) [reply products/BellHybrid/apps/application-bell-meditation-timer/windows/MeditationCountdownWindow.cpp]: done

File: products/BellHybrid/apps/application-bell-meditation-timer/windows/MeditationCountdownWindow.hpp (hunk 2 above)
Comment (Maciej-Mudita) [reply products/BellHybrid/apps/application-bell-meditation-timer/windows/MeditationCountdownWindow.hpp]: done

**Review by patrycja-paczkowska — APPROVED** [review]

**Pull request merged**